sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -199,7 +172,7 @@ def queue_size(self):
         return 0
 
     async def receive(self, *, max_batch_size=None, timeout=None):
-        # type: (int, float) -> List[EventData]
+        # type: (Any, int, float) -> List[EventData]
```

Comment:
Do we need this 'Any' in the type description? as the `*` is just for omitting positional args..we simply ignore position args.

Reply:
Without "Any" Pycharm would report type hint error and have funny code assistant.